Grading at 12/09/2022:19:26:14.183425
For commit ID 8bf952ce6718bf405db245a4edd82e5b2672620c
Grading at Fri Dec  9 02:25:50 PM EST 2022
Doing some basic checking for your submission

Step 1
Checking that cyoa-step1 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4
g++ -pedantic -g --std=gnu++98 -Werror -Wall -o cyoa-step1 cyoa-step1.cpp
The code compiled (good)
Executable cyoa-step1 exists (good)
------------------------------------
Now running YOUR tests for step 1
------------------------------------


1.1: Current test is ./cyoa-step1 story1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
  - Valgrind was clean (no errors, no memory leaks)
Valgrind was clean


1.2: Current test is ./cyoa-step1 story2 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
  - Valgrind was clean (no errors, no memory leaks)
Valgrind was clean


1.3: Current test is ./cyoa-step1 story4 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid line format in story.txt: 
2@kjhghj:page2.txt
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.4: Current test is ./cyoa-step1 story6 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Cannot open file: 
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


1.5: Current test is ./cyoa-step1 story8 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Cannot open file: story8/    page1.txt
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 1

Step 2
Checking that cyoa-step2 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4
g++ -pedantic -g --std=gnu++98 -Werror -Wall -o cyoa-step2 cyoa-step2.cpp
The code compiled (good)
Executable cyoa-step2 exists (good)
------------------------------------
Now running YOUR tests for step 2
------------------------------------


2.1: Current test is ./cyoa-step2 story4  <  story4/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid line format in story.txt: 
2@kjhghj:page2.txt
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.2: Current test is ./cyoa-step2 story6  <  story6/inputs/input5.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Cannot open file: 
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.3: Current test is ./cyoa-step2 story7  <  story6/inputs/input5.txtb
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
/dev/fd/3: line 528: story6/inputs/input5.txtb: No such file or directory
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


2.4: Current test is ./cyoa-step2 story8  <  story6/inputs/input5.txtb
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
/dev/fd/3: line 528: story6/inputs/input5.txtb: No such file or directory
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 2

Step 3
Checking that cyoa-step3 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4
g++ -pedantic -g --std=gnu++98 -Werror -Wall -o cyoa-step3 cyoa-step3.cpp
The code compiled (good)
Executable cyoa-step3 exists (good)
------------------------------------
Now running YOUR tests for step 3
------------------------------------


3.1: Current test is ./cyoa-step3 story1 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
  - Valgrind was clean (no errors, no memory leaks)
Valgrind was clean


3.2: Current test is ./cyoa-step3 story2 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
  - Valgrind was clean (no errors, no memory leaks)
Valgrind was clean


3.3: Current test is ./cyoa-step3 story4 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid line format in story.txt: 
2@kjhghj:page2.txt
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.4: Current test is ./cyoa-step3 story6 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Cannot open file: 
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


3.5: Current test is ./cyoa-step3 story7 < /dev/null
-----------------------------
Your output looks good. Lets check valgrind
  - Valgrind was clean (no errors, no memory leaks)
Valgrind was clean


3.6: Current test is ./cyoa-step3 story8 < /dev/null
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Cannot open file: story8/    page1.txt
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 3

Step 4
Checking that cyoa-step4 compiles
rm -f *~ cyoa-step1 cyoa-step2 cyoa-step3 cyoa-step4
g++ -pedantic -g --std=gnu++98 -Werror -Wall -o cyoa-step4 cyoa-step4.cpp
The code compiled (good)
Executable cyoa-step4 exists (good)
------------------------------------
Now running YOUR tests for step 4
------------------------------------


4.1: Current test is ./cyoa-step4 story4  <  story3/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid line format in story.txt: 
2@kjhghj:page2.txt
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean


4.2: Current test is ./cyoa-step4 story5  <  story5/inputs/input1.txt
-----------------------------
Great: we both considered this an error
----here is what you printed on stderr---
Invalid number to convert: -3:Purchase a muffin.
-----------------------------------------
Lets check if it valgrinds:
Valgrind was clean
No request to check for intermediate deadline completion on step 4

You have passed the test cases YOU provided
If your tests are good, this is great. 
If your tests are insufficient, you should do much more testing

Overall Grade: NO GRADE YET
